modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go

```diff
@@ -62,36 +63,55 @@ func (p *CustomFilter) StatusCondition() *model.SelectCondition {
 	return condition
 }
 
-func (p *CustomFilter) MemberCondition(key string) (*model.SelectCondition, error) {
+type MemberCondition struct {
+	executorCondition *model.SelectCondition
+	creatorCondition  *model.SelectCondition
+}
+
+func (p *CustomFilter) MemberCondition() (MemberCondition, error) {
 	members, err := p.bdl.ListMembers(apistructs.MemberListRequest{
 		ScopeType: apistructs.ProjectScope,
 		ScopeID:   int64(p.InParams.ProjectID),
 		PageNo:    1,
 		PageSize:  500,
 	})
 	if err != nil {
-		return nil, err
+		return MemberCondition{}, err
 	}
 
-	condition := model.NewSelectCondition(key, cputil.I18n(p.sdk.Ctx, key), func() []model.SelectOption {
+	executorCondition := model.NewSelectCondition("executor", cputil.I18n(p.sdk.Ctx, "executor"), func() []model.SelectOption {
 		selectOptions := make([]model.SelectOption, 0, len(members)+1)
 		for _, v := range members {
 			selectOptions = append(selectOptions, *model.NewSelectOption(func() string {
-				if v.Nick != "" {
-					return v.Nick
-				}
-				if v.Name != "" {
-					return v.Name
-				}
-				return v.Mobile
+				return v.GetUserName()
 			}(),
 				v.UserID,
 			))
 		}
 		selectOptions = append(selectOptions, *model.NewSelectOption(cputil.I18n(p.sdk.Ctx, "choose-yourself"), p.sdk.Identity.UserID).WithFix(true))
 		return selectOptions
 	}())
-	return condition, nil
+	executorCondition.ConditionBase.Placeholder = cputil.I18n(p.sdk.Ctx, "please-choose-executor")
+
+	creatorCondition := model.NewSelectCondition("creator", cputil.I18n(p.sdk.Ctx, "creator"), func() []model.SelectOption {
+		selectOptions := make([]model.SelectOption, 0, len(members)+1)
+		for _, v := range members {
+			selectOptions = append(selectOptions, *model.NewSelectOption(func() string {
+				return v.GetUserName()
+			}(),
```

Comment:
no need to wrap a func() {}(), just v.GetUserName()

Reply:
done